### ISSUE TITLE
Exporting

### ISSUE THREAD

**Issue by mrrrau** [gurgeh/selfspy#2]
Do you plan to add some export functionality of the logs?

Thanks.

**Comment by gabrielflorit** [comment]
Hey @gurgeh any updates on this? I can try working on it - https://github.com/gurgeh/selfspy/blob/master/selfspy/stats.py#L322, correct?

**Comment by gurgeh** [comment]
@gabrielflorit Yes, that would be one place where it can be put. Show rows but in CSV format. If you do it, I will merge.
